### PR TITLE
Applab: Fix resizing with scaled visualization

### PR DESCRIPTION
**Pull request by islemaster**
The resize operation was not scaling correctly with the visualization.  Now it does, so the cursor sticks to the element corner as you resize it.

Fixes [Pivotal #104479902](https://www.pivotaltracker.com/story/show/104479902).

**Before**
resize-before

**After**
resize-after

### CODE REVIEW

File: apps/src/applab/designMode.js [hunk 1]
```diff
@@ -469,7 +469,38 @@ function makeDraggable (jqueryElements) {
...
+        // Wishing for a vector maths library...
+
+        // Customize motion according to current visualization scale.
+        var div = document.getElementById('divApplab');
+        var xScale = div.getBoundingClientRect().width / div.offsetWidth;
+        var yScale = div.getBoundingClientRect().height / div.offsetHeight;
+        var deltaWidth = ui.size.width - ui.originalSize.width;
+        var deltaHeight = ui.size.height - ui.originalSize.height;
+        var newWidth = ui.originalSize.width + (deltaWidth / xScale);
+        var newHeight = ui.originalSize.height + (deltaHeight / yScale);
```
Comment (joshlory) [inline apps/src/applab/designMode.js]: Worth de-duping against https://github.com/code-dot-org/code-dot-org/pull/4311/files#diff-7f9b30159330297b3b2915dcafb93087L495 down in `.draggable`?

Comment (islemaster) [inline apps/src/applab/designMode.js]: Sure.

Do we actually need to calculate xScale and yScale separately here?  We always lock aspect ratio, AFAIK.

**Comment by islemaster** [comment]
@trevorb @joshlory De-duped a little bit.  Thoughts?

**Comment by islemaster** [comment]
Before merging this, I need to test it in IE - just learned that we have other scaling issues in that browser.

**Comment by trevorb** [comment]
LGTM

**Pull request merged**